app/views/curation_concerns/file_sets/_single_use_link_rows.html.erb

```diff
@@ -0,0 +1,16 @@
+<% single_use_links.each_with_index do |link, index| %>
+  <% type = link.path =~ /downloads/ ? 'Download' : 'Show' %>
```

Comment:
I don't feel very strongly about this one, but I wonder if `type`, `url_helper` (line 3), the truncated download key (line 5), and the formatted-in-hours link expiry time (line 5) should be added as model methods in CC. That way we can cut down on logic in this partial and potentially re-use this information in other views (if we ever want to surface this in a dashboard).